pkg/controller/androidvariant/androidvariant_controller.go

```diff
@@ -83,6 +84,17 @@ func (r *ReconcileAndroidVariant) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
+	// Check if the namespace of the cr is in the APP_NAMESPACES environment variable provided to the operator or if it's in the operator namespace
+	operatorNamespace, err := k8sutil.GetOperatorNamespace()
```

Comment:
This, and the resulting error check immediately below, _could_ be part of the `util.IsValidAppNamespace()` function, since in all 3 controllers we're calling this just to pass the result to that function.